### DISCUSSION TITLE
[Help] Exporting text on a path is not working as expected with the 'pathAlign' property

### DISCUSSION THREAD

**Discussion by kiguri** [fabricjs/fabric.js#9335] [General]
I am experiencing cropping at the top of the text (on a path) when exporting the canvas with a high multiplier (at least 10 times).
The text is slightly cropped when **pathAlign = 'baseline'** and got more when **pathAlign = 'descender'**

Link to reproduce: https://codesandbox.io/s/wizardly-bessie-z4yjmv?file=/src/App.js

**pathAlign = baseline**

![exported-bl](https://github.com/fabricjs/fabric.js/assets/35722061/3cc28349-f4d0-46e0-ad07-f394ba35a623)

**pathAlign = descender**

![expoted-desc](https://github.com/fabricjs/fabric.js/assets/35722061/fd27b106-538b-4762-a877-139cc49ce24a)

**Comment by ShaMan123** [comment]
The is because the bounding box calcualtion is wrong and doesn't account for the text on the path.

**Comment by asturur** [comment]
Yes, the text on a path is being added gradually and measuring the bounding box with a bit more of accuracy needs to be done yet.
It won't include proper font measurments anyway but should fix for all the basic fonts.

I m thinking how to do it since long, but i didn't have time.
